packages/garbo/src/config.ts

```diff
@@ -98,6 +99,14 @@ function defaultTarget() {
 export const targettingMeat = () =>
   $monsters`cheerless mime executive`.includes(globalOptions.target);
 
+export const targettingItems = () =>
+  valueDrops(globalOptions.target) > meatDrop(globalOptions.target);
```

Comment:
Honestly we should probably export these from lib as well, I recognize I'm the one to have added `targettingMeat` but while we're cleaning it up we might as well